src/data/cassandra/impl/ExecutionStrategy.h

```diff
@@ -140,10 +140,11 @@ class DefaultExecutionStrategy {
     ResultOrErrorType
     writeSync(StatementType const& statement)
     {
-        counters_->registerWriteSync();
+        auto const startTime = std::chrono::steady_clock::now();
```

Comment:
If we record start time here, we also include the retry duration ,right?

Reply:
Yeah, duration for these counters is a total duration from start to end including retries